string robot_id
float32 x
float32 y
string action      # "ocupar" ou "liberar"
float64 timestamp  # hora do evento